ci/setup_tpm12_simulator.sh

```diff
@@ -63,8 +63,46 @@ fetch_simulator () {
 build_simulator () {
   cp -v "${SIMULATOR_SRC}/tpm/makefile-tpm" "${SIMULATOR_SRC}/tpm/Makefile"
   cd "${SIMULATOR_SRC}/tpm" && make -j4
+  cd "${SIMULATOR_SRC}/libtpm" && ./autogen
+  cd "${SIMULATOR_SRC}/libtpm" && ./configure
+  cd "${SIMULATOR_SRC}/libtpm" && make -j4
+}
+
+run_simulator () {
+  mkdir -pv "${BUILD_BASE}/NVRAM"
+  export TPM_PORT='6545'
+  export TPM_PATH="${BUILD_BASE}/NVRAM"
+  export TPM_SERVER_NAME='localhost'
+  export TPM_SERVER_PORT='6545'
+  export TCSD_TCP_DEVICE_PORT='6545'
```

Comment:
Should this be in run_tcsd? Possibly define 6545 as a constant elsewhere.

Reply:
Done.